Packages/jp.keijiro.minis/Runtime/Internal/MidiPort.cs

```diff
@@ -50,6 +51,14 @@ public MidiPort(int portNumber, string portName)
             }
 
             RtMidiDll.OpenPort(_rtmidi, (uint)portNumber, "RtMidi Input");
+            var desc = new InputDeviceDescription {
+                interfaceName = "Minis",
+                deviceClass = "MIDI",
+                product = _portName,
+            };
+            var device = MidiSystemWrangler.QueueDeviceAddition(desc);
+            device.claimed = true;
```

Comment:
The created device shouldn't be immediately claimed, it should go through the same `CheckClaimed` method used in `ProcessMessageQueue`.

Devices are not immediately created through this method since the message queue is processed on a background thread; they are deferred to the next input system update and use signalling values to communicate between the threads. Immediately marking the device as claimed may cause null reference errors, since the device object will not be set until that update has occurred.

To be fair, the check this would break on in `MidiChannel` *should* be for if a device is null and not for if the device is claimed, that's my bad. The claimed value is just to ensure no device objects get orphaned/leaked, in retrospect it shouldn't be necessary.